.github/workflows/package.yml

```diff
@@ -15,12 +15,17 @@ jobs:
         uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
         with:
           python-version: "3.13"
+          cache: pip
+          cache-dependency-path: requirements-build.txt
```

Comment:
Could you maybe add a one-line comment clue for people not familiar with this `cache` feature? Is it part of the security or just an optimization?